### PR TITLE
[hi] Localize immutable-infrastructure.md

### LINKED ISSUE #1585: Immutable Infrastructure for Hindi
# This issue is to initiate a new l10n Team

Hindi version - Immutable Insfrastructure 
link -  https://glossary.cncf.io/immutable-infrastructure/

### PR DESCRIPTION
**Pull request by Aryan-Singla**
Signed-off-by: ARYANis0001 [devendergupta4573@gmail.com]
### Describe your changes
Localize immutable-infrastructure.md in Hindi

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1585 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by ARYANis0001** [comment]
Hey @bishal7679 can you please elaborate more about what changes you are expecting from this PR so that in the future I can take care of then

**Comment by seokho-son** [comment]
> Hey @bishal7679 can you please elaborate more about what changes you are expecting from this PR so that in the future I will take care of them

Regarding `what changes you are expecting`

@ARYANis0001 I have a common comment on your PRs.
PTAL: https://github.com/cncf/glossary/pull/2605#issuecomment-1815774999

**Comment by ARYANis0001** [comment]
Hey @aj11anuj 
@kumarankit999
I have added some words to simplify it can you please review it

**Review by kumarankit999 — APPROVED** [review]
Thanks for Contributing this, lgtm!!

**Comment by ARYANis0001** [comment]
Hey @aj11anuj @bishal7679
I have added some words to simplify it can you please review it

File: content/hi/immutable-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+शीर्षक: अपरिवर्तनीय अवसंरचना (Immutable Infrastructure)
+status: Completed
+श्रेणी: संपत्ति
+टैग: ["बुनियादी ढांचा", "संपत्ति", ""]
+---
+
+अपरिवर्तनीय अवसंरचना का तात्पर्य कंप्यूटर अवसंरचना से है
+([वर्चुअल मशीन](/वर्चुअल-मशीन/, [कंटेनर](/कंटेनर/), नेटवर्क उपकरण)
```
Comment (ankur0904) [inline content/hi/immutable-infrastructure.md:9]:
```suggestion
([वर्चुअल मशीन](/वर्चुअल-मशीन/), [कंटेनर](/कंटेनर/), नेटवर्क उपकरण)
```

File: content/hi/immutable-infrastructure.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+जिसे एक बार तैनात करने के बाद बदला नहीं जा सकता।
+इसे एक स्वचालित प्रक्रिया द्वारा लागू किया जा सकता है जो अनधिकृत परिवर्तनों को अधिलेखित कर देती है
+एक ऐसी प्रणाली के माध्यम से जो पहले स्थान पर परिवर्तन की अनुमति नहीं देगी।
+कंटेनर अपरिवर्तनीय बुनियादी ढांचे का एक अच्छा उदाहरण हैं
+क्योंकि कंटेनरों में लगातार परिवर्तन केवल कंटेनर का एक नया संस्करण 
+बनाके या मौजूदा कंटेनर को उसकी छवि से पुनः बनाके ही किए जा सकते हैं
```
Comment (ankur0904) [inline content/hi/immutable-infrastructure.md:15]: In this paragraph we are talking about the container image
छवि -> "Image(Picture)"
you can replace with `छवि` to `इमेज`

**Review by ankur0904 — CHANGES_REQUESTED** [review]
Some sentences are not clear. Please make sure to review it carefully.
e.g. line 12, 19

**Comment by ARYANis0001** [comment]
@ankur0904 Thanks any other suggestions

**Review by ARYANis0001 — COMMENTED** [review]
Signed-off-by: ARYANis0001 [devendergupta4573@gmail.com]

**Review by aj11anuj — CHANGES_REQUESTED** [review]

**Comment by Aryan-Singla** [comment]
Hi @aj11anuj I have updated the suggestions

**Pull request closed without merging**